Weather App/UI/HomeVC/HomeView.swift

```diff
@@ -31,17 +31,12 @@ class HomeView : UIView {
     private lazy var windLabel = UILabel()
     private var conditionId = 800
    
-    private let normalFontSize: CGFloat = 30.0
-    private let downAttributes:  [NSAttributedString.Key : Any] =
-        [
-            .font : UIFont.systemFont(ofSize: 30.0 - 4, weight: .bold),
-            .foregroundColor : UIColor.white
-        ]
-    private let upAttributes :  [NSAttributedString.Key : Any]  =
-        [
-            .font : UIFont.systemFont(ofSize: 30.0 - 8),
-            .foregroundColor : UIColor.white
-        ]
+    private let normalFontSize: CGFloat = {
+        UIScreen.main.bounds.height > 600 ? 30 : 24
```

Comment:
I'm not sure I understand the logic here for deciding on the font size. Usually, you get font size from design.

Reply:
I changed it to fit better for smaller screen sizes, if font should not be changed I will look for different solutions, maybe changing the available space for the text, as the height goes smaller, so it resizes itself if the text can not fit?

Reply:
I removed it, no need for that, also yes the designer will give the font size and that should be used.